proposals/0016-simd-application-fees.md

```diff
@@ -0,0 +1,149 @@
+---
+simd: '0016'
+title: Application Fees (Write-lock fees)
+authors:
+  - Godmode Galactus (Mango Markets)
+  - Maximilian Schneider (Mango Markets)
+category: Fees
+type: Fees
+status: Draft
+created: 2022-12-23
+---
+
+## Problem
+
+According to the discussion on the following issue:
+https://github.com/solana-labs/solana/issues/21883
+
+During network congestion, many transactions in the network cannot be processed 
+because they all want to write lock similar accounts. 
+When a write lock on an account is taken by a transaction batch no other batch 
+can use this account in parallel, so only transactions belonging to a single batch 
+are processed correctly and all others are forwarded to the leader of the next slot. 
```

Comment:
i can't agree enough. :) that's why I'm so inclined to kill bankingstage's batching mechanism altogether.